core/sphinx/sphinx_benchmark_test.go

```diff
@@ -64,7 +60,7 @@ func benchNewNode(mynike nike.Scheme) *nodeParams {
 	}
 	n.publicKey, n.privateKey, err = mynike.GenerateKeyPair()
 	if err != nil {
-		panic(err)
+		panic("wtf")
```

Comment:
as fitting as `"wtf"` might be in this situation, the `err` is probably more useful?

Reply:
fixed